pyproject.toml

```diff
@@ -10,6 +10,8 @@ flask = "^1.1.2"
 python-dotenv = "^0.14.0"
 Flask-WTF = "^0.14.3"
 requests = "^2.25.0"
+gunicorn = "^20.0.4"
+pytest = "^6.2.3"
```

Comment:
pytest is already included in the dev-dependencies section below

Reply:
That purest line is now removed